.test-infra/jenkins/job_LoadTests_SideInput_Python.groovy

```diff
@@ -0,0 +1,187 @@
+                        side_input_type      : 'dict',
+                        access_percentage    : 99,
+                ]
+        ],
+        [
+                title          : 'SideInput Python Load test: 20gb-1kb-5workers-1window-first-iterable',
+                test           : 'apache_beam.testing.load_tests.sideinput_test',
+                runner         : CommonTestProperties.Runner.DATAFLOW,
+                pipelineOptions: [
+                        job_name             : 'load-tests-python-dataflow-batch-sideinput-3-' + now,
+                        project              : 'apache-beam-testing',
+                        region               : 'us-central1',
+                        temp_location        : 'gs://temp-storage-for-perf-tests/loadtests',
+                        publish_to_big_query : true,
+                        metrics_dataset      : datasetName,
+                        metrics_table        : 'python_dataflow_batch_sideinput_3',
+                        influx_measurement   : 'python_batch_sideinput_3',
+                        input_options        : '\'{' +
+                                '"num_records": 20000000,' +
+                                '"key_size": 100,' +
+                                '"value_size": 900}\'',
+                        num_workers          : 5,
+                        autoscaling_algorithm: 'NONE',
+                        side_input_type      : 'iter',
+                        access_percentage    : 1,
+                ]
+        ],
+        [
+                title          : 'SideInput Python Load test: 20gb-1kb-5workers-1window-iterable',
+                test           : 'apache_beam.testing.load_tests.sideinput_test',
+                runner         : CommonTestProperties.Runner.DATAFLOW,
+                pipelineOptions: [
+                        job_name             : 'load-tests-python-dataflow-batch-sideinput-4-' + now,
+                        project              : 'apache-beam-testing',
+                        region               : 'us-central1',
+                        temp_location        : 'gs://temp-storage-for-perf-tests/loadtests',
+                        publish_to_big_query : true,
+                        metrics_dataset      : datasetName,
+                        metrics_table        : 'python_dataflow_batch_sideinput_4',
+                        influx_measurement   : 'python_batch_sideinput_4',
+                        input_options        : '\'{' +
+                                '"num_records": 20000000,' +
+                                '"key_size": 100,' +
+                                '"value_size": 900}\'',
+                        num_workers          : 5,
+                        autoscaling_algorithm: 'NONE',
+                        side_input_type      : 'iter',
+                ]
+        ],
+        [
+                title          : 'SideInput Python Load test: 2gb-1kb-5workers-1window-first-list',
```

Comment:
Looks like even 10GB will be too much for lists. I get the following error:

```
Workflow failed. Causes: S03:Impulse+Collect start time metrics+ParDo(SequenceSideInputTestDoFn)/ParDo(SequenceSideInputTestDoFn)+Collect end time metrics failed., The job failed because a work item has failed 4 times. 
...
Root cause: The worker lost contact with the service.
```